docs/rules/eol-last.md

```diff
@@ -8,7 +8,8 @@ as output files to the terminal without interfering with shell prompts.
 
 ## Rule Details
 
-This rule requires at least one newline at the end of non-empty files.
+This rule enforces at least one newline (or absence thereof) at the end
```

Comment:
Could you update the title as well? "Require or disallow file to end with single newline"